integrationtest/mapper-pojo/src/test/java/org/hibernate/search/v6poc/integrationtest/mapper/pojo/JavaBeanMappingFailureReportIT.java

```diff
@@ -46,7 +46,7 @@
 
 	/**
 	 * Test mapping with failures in the same context
-	 * and check that every failure is reported, and that failures are grouped into a single list.
+	 * and convert that every failure is reported, and that failures are grouped into a single list.
```

Comment:
OK, I was reading the patch backwards and I'm out of patience :). You should **check** all the occurrences of `check` -> `convert` in this patch.

Reply:
Thanks, done.